pages/_app.tsx

```diff
@@ -1,8 +1,21 @@
-import "../styles/globals.css";
+import { Inter, Outfit } from "@next/font/google";
 import type { AppProps } from "next/app";
+import "../styles/globals.css";
+
+const inter = Inter({
+  variable: "--font-inter",
```

Comment:
Si especificas los `weight`  que se van a utilizar se podrian ahorrar un par de Kb (no se si nextjs haga eso en el build).
Esto se puede verificar desde el "network" tab en el DevTools

Reply:
@TonyMckes Gracias por el consejo Tony!
Por cierto, crees necesario especificar un `fallback`?

> Podrias encontrar el porque las fuentes no estan cargando correctamente en los textos que no estan envueltos en etiquetas de heading (h1, h2, etc..).
Pensé que era el comportamiento deseado, ya que la config especifica `font-inter` para el `html` y unicamente `font-outfit` para los headings (`h1, h2, etc`). Está bien o no debería ser así?

> Aparte, podrias revertir el ultimo commit (https://github.com/Juguetear/juguetear-web/commit/80db11d6e30a5aadfbdc9af1c003720799f06e04)?
Había olvidado ese comando!
Entendí como funciona, pero pensé que ser un archivo con extensión `js` no iba a funcionar.

Reply:
@TonyMckes Please confirmame respecto al revert y si el import del type funciona a pesar de que sea otro tipo de extensión.

Reply:
> crees necesario especificar un `fallback`?

No, no creo que haga falta

> Pensé que era el comportamiento deseado, ya que la config especifica `font-inter` para el `html` y unicamente `font-outfit` para los headings (`h1, h2, etc`). Está bien o no debería ser así?

Si, es el comportamiento esperado. Pero como que las `Inter` no estan renderizando, aunque si cargan eh
~~I can tell you how to fix it if need it~~

> Please confirmame respecto al revert y si el import del type funciona a pesar de que sea otro tipo de extensión.

Ese tipo de imports es como para agregar IntelliSense a archivos JavaScript
![image](https://user-images.githubusercontent.com/78808163/208424859-aa498b18-bf8c-4e7c-9b1d-71b2ad75d181.png)